client/src/components/TabBody/style.css

```diff
@@ -0,0 +1,35 @@
+
+.tabBody {
+  display: flex;
```

Comment:
You don't need to use flex here 
All element you are using is block

Reply:
Deleted